### ISSUE TITLE
APRS HF (300 Baud?)

### ISSUE THREAD

**Issue by VA7BDT** [ge0rg/aprsdroid#253]
Hi, I am not sure if this has been asked before, but I am not a programmer and also brand new to the HAM radio world.  As a result I recently got APRSDroid working on VHF and find this application totally AWESOME!  I am curious if there is some way some how to set the baud rate to 300baud so that it can be used over HF?  Unless I am mistaken, this should be the only change required?  I was able to setup a test on 50Mhz with a friend who setup a digipeater and we were able to update APRS Locations using APRSDroid.  Any advice / suggestions/ guidance would be greatly appreciated for a newby.

Again, AWESOME APP, its a shame more people don't appreciate the work that went into making it!

**Comment by a8ksh4** [comment]
What kind of TNC are you using?  I think this would be a tnc capability, so if you can find a bluetooth ip tnc that supports 300 baud, it would work fine with aprsdroid.  E.g. direwolf tnc on pi zero acting as tnc for your phone.

**Comment by ge0rg** [comment]
Yeah, the AFSK code in APRSdroid is rather legacy (and a third-party contribution), so I'm not really sure how to change it to work with 300bd. If you can get hold of a TNC that does 300bd, you can connect it via Bluetooth Serial, via TCP/IP or via USB to your Android and it should work.
